lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -169,4 +172,19 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
       ie -> if InformedEntity.parent_station?(ie), do: ie.stop
     end)
   end
+
+  defp has_nearby_redundancy?(
+         %Alert{informed_entities: [%{facility: %{id: informed_facility_id}} | _]},
+         redundancy_data
+       ) do
+    Enum.find(redundancy_data, &(&1.id == informed_facility_id))[:nearby_redundancy?]
```

Comment:
* It's possible the ID we're looking for doesn't exist in the committed data (`find` may return `nil`), but we're not handling that case here.
* We could instead structure the data as a map where the elevator IDs are keys (`{"814": {"nearby_redundancy?": true}, ...}`), so instead of doing a linear-time search through the entire list for every active elevator closure, we could use a map lookup.

Reply:
Good idea. I feel like Betsy and Dave would want to know about elevators missing from the map so will ask them what they want to happen when that occurs. Might just add a log and return `false` in that case.